aplib.net-demo/Assets/Scripts/WFC/GridPlacer.cs

```diff
@@ -115,7 +144,7 @@ public void Awake()
         /// Makes the scene.
         /// </summary>
         /// <exception cref="Exception">The amount of rooms is larger than the available places in the grid.</exception>
-        public void MakeScene()
+        private void MakeScene()
```

Comment:
Nitpicky: In our style guide it says all public methods should be above the private ones.